frontend/packages/dev-console/src/components/pipelines/__tests__/PipelineRun.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { ListPage } from '@console/internal/components/factory';
 import { shallow } from 'enzyme';
-import PipelineRuns from '../PipelineRuns';
+import PipelineRuns from '../detail-page-tabs/PipelineRuns';
```

Comment:
Did you just move the test away from the component it was testing?

Why? What organization structure does this serve?

Reply:
have put it back to where it belongs